src/psmove.c

```diff
@@ -303,6 +310,66 @@ static int psmove_remote_disabled = 0;
 static int psmove_num_open_handles = 0;
 
 
+static int
+psmove_fscanf_c(FILE *fp, const char *fmt, ...)
+{
+    int result = -1;
+
+    va_list args;
+    va_start(args, fmt);
+#if defined(_WIN32) && defined(_UCRT)
+    /* Locale APIs only available with the Universal CRT */
+    _locale_t loc = _create_locale(LC_NUMERIC, "C");
+    result = _vfscanf_l(fp, fmt, loc, args);
+    _free_locale(loc);
+#elif defined(_WIN32)
+    result = vfscanf(fp, fmt, args);
+#else
```

Comment:
Unsurprisingly, MSVCRT also needs the locale treatment. So this special case will result in the program misbehaving on those platforms.

Reply:
This is taken care of in d37b68d523bf20b53b6da49e18dd9a381063357e -- if it works :)